base/src/test/java/io/spine/io/EnsureSpec.java

```diff
@@ -47,9 +47,9 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @DisplayName("`Ensure` utilities class should")
-class EnsureTest extends UtilityClassTest<Ensure> {
+class EnsureSpec extends UtilityClassTest<Ensure> {
```

Comment:
Let's not rename all of our Java tests into "...Spec". Otherwise, it should be done for each and every test class in the framework. Since it's not a Kotlin test type, I don't see any reason for such a massive change.